server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -47,6 +53,35 @@ impl RemoteDataSynchroniser {
         Ok(())
     }
 
+    /// Request site info and persist it
+    pub(crate) async fn request_and_set_site_info(
+        &self,
+        connection: &StorageConnection,
+    ) -> Result<(), RequestAndSetSiteInfoError> {
+        info!("Requesting site info");
+        let site_info = self
+            .sync_api_v5
+            .get_site_info()
+            .await
+            .map_err(RequestAndSetSiteInfoError::RequestSiteInfoError)?;
+
+        self.set_site_info(&connection, site_info)
+            .map_err(RequestAndSetSiteInfoError::SetSiteInfoError)?;
+
+        info!("Received site info");
+        Ok(())
+    }
+
+    pub(crate) fn set_site_info(
```

Comment:
can't this be private? and if yes just inline these three lines?

Reply:
I like it, thanks!

https://github.com/openmsupply/open-msupply/pull/388/commits/06b0fbcfdb5d9a627b9532ff45984d9585da5bc6